core/stencil.config.ts

```diff
@@ -257,4 +257,8 @@ export const config: Config = {
   globalScript: 'src/global/ionic-global.ts',
   enableCache: true,
   transformAliasedImportPaths: true,
+  extras: {
+    experimentalSlotFixes: true,
+    experimentalScopedSlotChanges: true,
```

Comment:
Let's add a comment noting that these are needed until a stable version is released.